src/Swarm/Game/State.hs

```diff
@@ -842,6 +855,7 @@ activateRobot rid = internalActiveRobots %= IS.insert rid
 wakeUpRobotsDoneSleeping :: Has (State GameState) sig m => m ()
 wakeUpRobotsDoneSleeping = do
   time <- use ticks
+  
```

Comment:
Is the `<<.= Nothing` on the next line equivalent to:
```
internalWaitingRobots %= delete time
```
?

Wondering how that works, since `internalWaitingRobots` doesn't have `Maybe` values.